src/commands/helm-deploy.yml

```diff
@@ -0,0 +1,98 @@
+description: >
+  Deploys Helm to ShuttleCrawler, assumes image version of component is set in .version
+parameters:
+  app-name:
+    type: string
+    description: Name of application to be deployed
+  environment:
+    type: string
+    description: Environment to be deployed on (prod, staging, dev)
+  component:
+    type: string
+    description: Name of folder which contains the Helm Chart
+  domain:
+    type: string
+    description: Domain (K8s namespace) to deploy to
+  path:
+    type: string
+    description: Path to .chart folder of application
+  short-region:
+    type: string
+    default: usc1
+    description: Short-region to deploy Compute Engine resources
+  region:
+    type: string
+    default: us-central1
+    description: Region to deploy Compute Engine resources
+  registry-region:
+    type: string
+    default: us-central1
+    description: Region of Artifact Registry
+  dry-run:
+    type: boolean
+    default: false
+    description: If enabled, simulate an upgrade
+  channel:
+    type: string
+    description: Slack channel to send notifications to
+    default: deploys-staging
+steps:
+- checkout
+- gcp-authorize:
+    environment-variable-name: GCLOUD_SERVICE_ACCOUNT
+    do-helm-login: 'true'
+- run:
+    name: Setting up ShuttleCrawler Google and K8 context
+    command: |
+      gcloud container clusters get-credentials \
+        platform-<< parameters.environment >>-<< parameters.region >>-cluster \
+        --zone << parameters.region >> \
+        --project ${GCLOUD_PROJECT_ID}
```

Comment:
wait, doesn't gcp-authorize + do-helm-login do this already?

Reply:
I'm not sure exactly why I added this, but if I remove this part from monorepo helm-deploy, it errors saying "Kubernetes cluster unreachable".  I'll look into this further since it seems like cloud-router doesn't use this.

Reply:
OHhhhhhhhhhhhhhhhhhhhhhhhh

I'd be a fan of adding that as something the `do-helm-login` parameter causes in the `gcp-authorize` command

Reply:
hmm that would require gcp-authorize to take in extra params though like region, cluster name, project id

Reply:
hmmmmmmmmmmmmmm yes. Maybe a `gcp-kubectl-authorize` command then, but uck

Reply:
`gcp-kubectl-authorize` inside of internal-platform-orb?